ui/eureka/eureka.js

```diff
@@ -3082,12 +3832,12 @@ class EurekaTracker {
         this.PlaySound(this.options.BunnyPopSound, this.options.BunnyPopVolume);
     } else if (fate.isCritical) {
       const shouldPlay = fate.isDuelPrecursor && this.options.PopNoiseForDuel ||
-          this.options.PopNoiseForCriticalEngagement;
+        this.options.PopNoiseForCriticalEngagement;
```

Comment:
Can you undo these whitespace changes? In general I think lines that wrap should be indented with 4 spaces instead of 2 to differentiate them from a block indent.

Reply:
Well it was applied by ESLint, and it should follow the rules you established ? I'll undo those I can find, sure !

Reply:
It's done !